src/components/ProposalTable.vue

```diff
@@ -0,0 +1,308 @@
+<template>
+	<a-row>
+		<a-col :xs="{ offset: 1 }" :lg="{ offset: 15 }">
+			<a-radio-group
+				v-model:value="this.proposalStore.filterType"
+				name="radioGroup"
+				@change="handleFilter"
+			>
+				<a-radio :value="ProposalFilterType.All">{{
+					ProposalFilterType[ProposalFilterType.All]
+				}}</a-radio>
+				<a-radio :value="ProposalFilterType.Active">{{
+					ProposalFilterType[ProposalFilterType.Active]
+				}}</a-radio>
+				<a-radio :value="ProposalFilterType.Future">{{
+					ProposalFilterType[ProposalFilterType.Future]
+				}}</a-radio>
+				<a-radio :value="ProposalFilterType.Past">{{
+					ProposalFilterType[ProposalFilterType.Past]
+				}}</a-radio>
+			</a-radio-group>
+		</a-col>
+	</a-row>
+	<div class="tableContainer">
+		<a-table
+			:dataSource="dataSource"
+			:columns="columns"
+			bordered
+			:pagination="false"
+		>
+			<template #title>
+				<a-col>
+					<h3 style="text-align: center">Available Proposals</h3>
+				</a-col>
+			</template>
+			<template #bodyCell="{ record, column }">
+				<template v-if="column.key === 'name'">
+					<a :href="'//' + record.url" target="_blank">
+						{{ record.name }}
+					</a>
+				</template>
+				<template v-if="column.key === 'voting_start'">
+					{{
+						secToFormat(record.voting_start, "lll") +
+						" - " +
+						secToFormat(record.voting_end, "lll")
```

Comment:
can we change it to `formatSecondsWithType`

Reply:
There is no type here.

Reply:
`"lll"` is the format type

Reply:
Done